Links.md

```diff
@@ -0,0 +1,75 @@
+# Link encoding in IPLD
+
+*Draft* notes and recommendations for link encoding and decoding across IPLD
+codec implementations.
+
+## Terms
+
+A logical separation exists in any given IPLD codec between the **format** and the **serializer/deserializer**.
+
+```
++--------------------+             +---------------------+
+|                    |             |                     |
+|     Serializer     |             |    Deserializer     |
+|                    |             |                     |
++---------+----------+             +----------+----------+
+          |                                   ^
+          |         Sent to another peer      |
+          v                                   |
++---------+----------+             +----------+----------+
+|                    |             |                     |
+|      Format        +------------->       Format        |
+|                    |             |                     |
++--------------------+             +---------------------+
+```
+
+A codec may represent object types and tree structures any way it wishes.
+This includes existing representations (JSON, BSON, CBOR, Protobuf, msgpack,
+etc) or even new custom serializations. We will refer to this as the
+**representation**.
+
+Therefor, a **format** is the standardized representation of IPLD Links and Paths in a given **representation**.
```

Comment:
Currently, the "format" is describes how to go translate between structured data and binary.